js/cmp/post-comments.js

```diff
@@ -10,21 +10,37 @@ import LoadMoreComments from './load-more-comments.js'
 import PostPreview from './post-preview.js'
 import { state } from '../index.js'
 import { setOpen } from '../actions.js'
+import { SortingOptions } from '../util.js'
 
 const PostComments = ({ attrs: { post } }) => {
   let comments = []
   let loading = true
+
   // load comments
-  getComments(post).then(data => {
-    comments = data
-    loading = false
-    m.redraw()
-  })
+  const loadComments = sort => {
+    getComments(post, null, sort).then(data => {
```

Comment:
shouldn't we set `loading = true` when `loadComments` is called to indicate the state on the screen?
```suggestion
    loading = true
    getComments(post, null, sort).then(data => {
```

Reply:
Fixed this